doc/classes/CodeEdit.xml

```diff
@@ -500,20 +500,20 @@
 		<method name="unfold_all_lines">
 			<return type="void" />
 			<description>
-				Unfolds all lines, folded or not.
+				Unfolds all lines that are folded.
 			</description>
 		</method>
 		<method name="unfold_line">
 			<return type="void" />
 			<param index="0" name="line" type="int" />
 			<description>
-				Unfolds all lines that were previously folded.
+				Unfolds the given line if it is folded or if it is hidden under a folded line.
 			</description>
 		</method>
 		<method name="unindent_lines">
 			<return type="void" />
 			<description>
-				Unindents selected lines, or in the case of no selection the caret line by one. Same as performing "ui_text_unindent" action.
+				Unindents all lines that are selected or have a caret on them. Uses spaces or a tab depending on [member indent_use_spaces]. Equivalent to performing the [member ProjectSettings.input/ui_text_dedent] action. See [method indent_lines].
```

Comment:
"activating" sound a little repetitive with the "action" after it. Also its a shortcut, so "performing a shortcut" feels better than "activating a shortcut" to me. 
Actually, "using a shortcut" is probably better, I can change both to be more consistent with "using".

Reply:
Not particularly fond of it, personally. Someone else may beg to differ. "using" is way too generic.

Reply:
How about "Equivalent to the [member ProjectSettings.input/ui_text_dedent] action"? I don't think a verb there is actually needed.
And I can reword `do_indent` to be consistent.

Reply:
Good point. In this case it's like saying "[This method is] equivalent to the [member ProjectSettings.input/ui_text_dedent] action". Although one could argue it's ambiguous since one is a method and one is a property, but... n-eh...